api/orchestrator/orchestrator.go

```diff
@@ -62,6 +63,12 @@ type CloudServiceRequest interface {
 	proto.Message
 }
 
+type LogRequest interface {
+	GetId() string
+	GetType() string
+	GetCloudServiceId() string
```

Comment:
I would omit that here and check if the request fullfills `CloudServiceRequest` in the log message code.

Reply:
That also bothered me all the time :)